cv_next/app/cv/[cvId]/components/commentSection/commentForm.tsx

```diff
@@ -0,0 +1,59 @@
+"use client";
+import { addComment } from "@/app/actions/comments/addComment";
+import { useSupabase } from "@/hooks/supabase";
+import { useRef } from "react";
+import { RxPaperPlane } from "react-icons/rx";
+import { mutate } from "swr";
+
+const COMMENT_FIELD_NAME = "comment";
+
+export default function CommentForm({ cv }: { cv: CvModel }) {
+  const formRef = useRef<HTMLFormElement | null>(null);
+  const supabase = useSupabase();
+
+  const formAction = async (formData: FormData) => {
+    const userId = (await supabase.auth.getUser()).data.user?.id;
+    if (!userId) throw new Error("User not found"); // TODO: handle this
+    const comment: NewCommentModel = {
+      data: formData.get(COMMENT_FIELD_NAME) as string,
+      document_id: cv.id,
+      parent_comment_Id: null,
+      user_id: userId,
+    };
+
+    await addComment(comment).finally(() => {
+      formRef.current?.reset();
+      mutate(cv.id);
+    });
+  };
+
+  return (
+    <form className="mb-6" ref={formRef} action={formAction}>
+      <div
+        className="grid grid-cols-[90%_10%] rounded-lg rounded-t-lg border border-gray-200 bg-white py-2
+                            pl-4 pr-2 dark:border-gray-700 dark:bg-gray-800"
+      >
+        <label htmlFor={COMMENT_FIELD_NAME} className="sr-only">
+          Your comment
+        </label>
+        <textarea
+          id={COMMENT_FIELD_NAME}
```

Comment:
This doesn't clash when multiple forms are open? You have a single id for all forms (if there's a possibility to open more than one).

Reply:
Only one form is possible